Code/Framework/AzFramework/AzFramework/Windowing/WindowBus.h

```diff
@@ -127,6 +152,9 @@ namespace AzFramework
 
         //! This is called once when the window is Activated and also called if the user resizes the window.
         virtual void OnWindowResized(uint32_t width, uint32_t height) { AZ_UNUSED(width); AZ_UNUSED(height); };
+        
+        //! This is called when the window's desired render resolution is changed.
+        virtual void OnResolutionChanged(uint32_t width, uint32_t height) { AZ_UNUSED(width); AZ_UNUSED(height); };
```

Comment:
I'm totally good with this API addition, but I just want to point out for consideration that the usage below seemed like OnWindowResized / OnResolutionChanged are typically called in pairs. If any listeners end up needing to know about both of these, they might need to create some workarounds or implicit assumptions about the calling order so that they only process once they have complete information from both notifications.

I think in this case though, the listeners that I saw below in the PR were only listening for one or the other, not both, which is why I don't think it's _actually_ a problem in this case. It's just a tricky thing to watch out for with API design, so I wanted to call it out for consideration in case you ran into any of this yourself while making these changes.

Reply:
Currently, the implementation is that when both values are changed, the notifications would be called after both values are changed. So if people is trying to get render resolution in OnWindowResized() function, they would get the latest the render solution (the one paired with window size). 

There are use cases that the two values are not changed together. For example, when MPS's full screen states change, the window size is changed, but not the render resolution, in this case, we don't need to handle swapchain resize (for dx12).